### PR TITLE
Switch to minecraftstatus library for !!mcstatus command

### PR DESCRIPTION
**Pull request by Infernum1**
- Use `!!minecraftstatus` instead of bare api requests for the server card and status.
- Tested on a test environment. Would suggest to test on the test bot.

image

### CODE REVIEW

File: bot/cogs/commands/minecraft.py [hunk 1]
```diff
@@ -228,22 +218,20 @@ async def mcstatus(self, ctx: Ctx, host=None, port: int = None):
...
 
-        embed.set_image(
-            url=f"https://api.iapetus11.me/mc/server/status/{combined}/image?v={random.random()*100000}"
-        )
-
-        if jj["favicon"] is not None:
+        card = discord.File(server_card, "server_card.png")
+        embed.set_image(url="attachment://server_card.png")
+        if server.favicon is not None:
             embed.set_thumbnail(
```
Comment (Infernum1) [inline bot/cogs/commands/minecraft.py:230]: currently don't have a method to get the favicon (through url) to set as thumbnail

File: bot/cogs/commands/minecraft.py [hunk 2]
```diff
@@ -155,50 +156,39 @@ async def mcstatus(self, ctx: Ctx, host=None, port: int = None):
 
             combined = f"{host}{port_str}"
 
-        fail = False
-        jj: dict = None
-
-        async with SuppressCtxManager(ctx.typing()):
```
Comment (Iapetus-11) [inline bot/cogs/commands/minecraft.py:161]: Why was this `SuppressCtxManager(ctx.typing())` removed? I think it's nice to show that the bot is "thinking" (while fetching the status)

File: bot/cogs/commands/minecraft.py [hunk 3]
```diff
@@ -156,23 +156,24 @@ async def mcstatus(self, ctx: Ctx, host=None, port: int = None):
...
-                server_card = await client.get_server_card(combined.replace("/", "%2F"))
-
-            except minecraftstatus.errors.ServerNotFound:
-                await ctx.reply(
-                    embed=discord.Embed(
-                        color=self.bot.embed_color,
-                        title=ctx.l.minecraft.mcping.title_offline.format(
-                            self.d.emojis.offline, combined
+        async with SuppressCtxManager(ctx.typing()):
+            async with minecraftstatus.MCStatus() as client:
```
Comment (Iapetus-11) [inline bot/cogs/commands/minecraft.py]: You can have context managers on the same line fwiw 👁️ 

`async with x() as y, a() as b:`

Comment (Infernum1) [reply bot/cogs/commands/minecraft.py]: oh damn, thanks!

File: bot/cogs/commands/minecraft.py (hunk 2 above)
Comment (Iapetus-11) [reply bot/cogs/commands/minecraft.py:161]: https://github.com/Iapetus-11/Villager-Bot/pull/277/commits/97f04e3a9e55ebfc5e3b8f38955b9f18102099a8

File: bot/cogs/commands/minecraft.py (hunk 3 above)
Comment (Iapetus-11) [reply bot/cogs/commands/minecraft.py]: https://github.com/Iapetus-11/Villager-Bot/pull/277/commits/dc60e23cb6a7f5fec506a67b38e7d6f559a24f54

**Review by Iapetus-11 — APPROVED** [review]
lgtm

**Pull request merged**